config/admin-functions/mock-vx-certifier.sh

```diff
@@ -20,6 +20,7 @@ if [[
     "${VX_MACHINE_TYPE}" != "admin" &&
     "${VX_MACHINE_TYPE}" != "central-scan" &&
     "${VX_MACHINE_TYPE}" != "mark" &&
+    "${VX_MACHINE_TYPE}" != "mark-scan" &&
```

Comment:
There are a few cert-related tweaks we need to make to vxsuite, too, e.g., here:

https://github.com/votingworks/vxsuite/blob/95c277b101c74aa1e1ee5490bb77c7f5e7cdaae4/libs/auth/src/certs.ts#L140-L149

I can open a PR for those!

Reply:
https://github.com/votingworks/vxsuite/pull/4604